test/dotnet-new.Tests/NewCommandTests.cs

```diff
@@ -16,7 +16,7 @@ public void WhenSwitchIsSkippedThenItPrintsError()
 
             cmd.ExitCode.Should().NotBe(0);
             
-			cmd.StdErr.Should().Be("No templates matched the input template name: [Web1.1]\n");
+			cmd.StdErr.Should().StartWith("No templates matched the input template name: [Web1.1]\n");
```

Comment:
This won't properly match "....\r\n"
Perhaps just remove the newline from the check?